src/test/kotlin/com/igorwojda/integer/power/challenge.kt

```diff
@@ -2,9 +2,10 @@ package com.igorwojda.integer.power
 
 import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
+import kotlin.math.pow
 
 private fun power(base: Int, exponent: Int): Int {
-    TODO("not implemented")
+    return base.toDouble().pow(exponent.toDouble()).toInt()
```

Comment:
Challenge: reimplement this by using the following concepts:
1. List initializer: https://kotlinlang.org/docs/reference/constructing-collections.html#initializer-functions-for-lists
2. Reduce operation: https://kotlinlang.org/docs/reference/collection-aggregate.html#fold-and-reduce
3. An operator: https://kotlinlang.org/docs/reference/operator-overloading.html
4. A method reference (to the above operator): https://khan.github.io/kotlin-for-python-developers/#function-references

It should be a one-liner ;)